src/Console/Command/LevelsCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Console\Command;
+
+use Rector\Console\ConsoleStyle;
+use Rector\Naming\CommandNaming;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+
+final class LevelsCommand extends Command
+{
+    /**
+     * @var ConsoleStyle
+     */
+    private $consoleStyle;
+
+    public function __construct(ConsoleStyle $consoleStyle)
+    {
+        $this->consoleStyle = $consoleStyle;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->setName(CommandNaming::classToName(self::class));
```

Comment:
Why not something like `--levels`? Like `--help`? Or no? :)

Reply:
It's common approach in CLI command to name them.

Like 

```
git clone ...
bin/rector process ...
bin/rector levels
```

Options you used for to extend command. Help is exception in this.

Reply:
Is that what you meant?

Reply:
Yes, but good explanation, TIL